package/src/store/apis/getChannels.ts

```diff
@@ -40,9 +40,9 @@ export const getChannels = <
   // Enrich the channels with state
   return channels.map((c) => ({
     ...mapStorableToChannel<StreamChatGenerics>(c),
-    members: cidVsMembers[c.cid],
-    messages: cidVsMessages[c.cid],
+    members: cidVsMembers[c.cid] || [],
```

Comment:
Since the handling is the same everywhere, we add a default value for good measure.